### ISSUE TITLE
JavaScript agent does not validate code

### ISSUE THREAD

**Issue by mrcdb** [huginn/huginn#2811]
I have noticed that the JS agent does not validate the code that is interactively set from the config page. Is this behaviour normal? I can replicate the issue by inserting random text into the textbox and applying the new configuration.

 I am using the latest Huginn Docker release from Docker hub.

Thanks in advance!

**Comment by dsander** [comment]
That is correct, I don't know if the library we are using support validating the syntax without running the code. Even if it does one would still need to (dry) run the Agent to ensure there are no logic errors.
